src/figuresClasses.ts

```diff
@@ -1,19 +1,114 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
Make [enums](https://www.typescriptlang.org/docs/handbook/enums.html) for color and shape, so you don't need to deal with bare strings